R/local.R

```diff
@@ -101,6 +102,7 @@ local_reproducible_output <- function(width = 80,
     cli.dynamic = FALSE,
     cli.unicode = unicode,
     cli.condition_width = Inf,
+    cli.num_colors = if (crayon) 8L else 0L,
```

Comment:
I would set it to 1 instead of 0. 0 might work, but nevertheless we usually use 1:
```r
❯ withr::with_envvar(c(NO_COLOR = "true"), cli::num_ansi_colors())
[1] 1
```

Reply:
Done, thanks!